AdyenCard/Utilities/Encryption/Extensions/StringExtension.swift

```diff
@@ -5,29 +5,18 @@
 //
 
 import CommonCrypto
+import Foundation
 
 extension String {
 
-    internal func sha1() -> String {
```

Comment:
Why this is removed?

Reply:
its not used :-) , right?

Reply:
ok, good point! Indeed it is no longer used!